ui/apps/platform/src/Containers/Clusters/NoClustersPage.tsx

```diff
@@ -185,19 +186,12 @@ function NoClustersPage({ isModalOpen, setIsModalOpen }): ReactElement {
                                 Legacy installation method
                             </Link>
                             {initBundlesCount !== 0 && (
-                                <Text component="p">
-                                    If you lost the YAML file that you downloaded,{' '}
-                                    <Link
-                                        to={`${clustersInitBundlesPath}?action=create`}
-                                        onClick={() => {
-                                            analyticsTrack({
-                                                event: CREATE_INIT_BUNDLE_CLICKED,
-                                                properties: { source: 'No Clusters' },
-                                            });
-                                        }}
-                                    >
-                                        create another init bundle
-                                    </Link>
+                                <Text component="p" className="pf-u-w-50vw">
+                                    If you misplaced your init bundle, we recommend locating the
+                                    previously downloaded YAML on your device first by the name of
+                                    the{' '}
+                                    <Link to={clustersInitBundlesPath}>generated init bundle</Link>,
```

Comment:
Did you mean to remove the analytics tracking click?

Reply:
yeah i don't think there's an event for the list of bundles -- that analytics link was just for when the user wants to create a new bundle if i am understanding correctly